integrations/postcss/next.test.ts

```diff
@@ -126,26 +127,35 @@ test(
         `,
       },
     },
-    async ({ fs, spawn, getFreePort }) => {
-      let port = await getFreePort()
-      await spawn(`pnpm next dev ${bundler === 'turbo' ? '--turbo' : ''} --port ${port}`)
+    async ({ fs, spawn, expect }) => {
+      let process = await spawn(`pnpm next dev ${bundler === 'turbo' ? '--turbo' : ''}`)
+
+      let url = ''
+      await process.onStdout((m) => {
+        let match = /Local:\s*(http.*)/.exec(m)
+        if (match) url = match[1]
+        return Boolean(url)
+      })
```

Comment:
We can potentially abstract this in a function but the regex is different based on the project. It also depends on the project if stdout or stderr is used. Because this is only a handful of lines, I just duplicated them.

Reply:
Seems fine to me.